src/object.d

```diff
@@ -3241,6 +3241,268 @@ template RTInfo(T)
     enum RTInfo = null;
 }
 
```

Comment:
Please add a regular (not documentation) comment stating that this function is called by the lowering of ordering comparisons.